packages/grid/_modules_/grid/components/tools/Preferences.tsx

```diff
@@ -0,0 +1,141 @@
+import { ClickAwayListener, Paper, Popper, Tab, Tabs, Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import * as React from 'react';
```

Comment:
I think that we should have nested imports to minimize size in dev mode.

```suggestion
import * as React from 'react';
import { makeStyles, Theme } from '@material-ui/core/styles';
import ClickAwayListener from '@material-ui/core/ClickAwayListener';
import Paper from '@material-ui/core/Paper';
import Popper from '@material-ui/core/Popper';
import Tab from '@material-ui/core/Tab';
import Tabs from '@material-ui/core/Tabs';
```